F/FlowCutterPACE17/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FlowCutterPACE17"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/kit-algo/flow-cutter-pace17.git", "fd405a2b003c9a93838e0488728addc0610a769f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd flow-cutter-pace17/
+$CXX -c -Wall -std=c++11 -O3 -DNDEBUG src/*.cpp
+mkdir -p "${bindir}"
+$CXX -o "${bindir}/flow_cutter_pace17${exeext}" *.o
+install_license LICENSE
```

Comment:
Can simplify even further (in this case the license should be installed automatically):
```suggestion
cd $WORKSPACE/srcdir/flow-cutter-pace17/
mkdir -p "${bindir}"
c++ -o "${bindir}/flow_cutter_pace17${exeext}" -Wall -std=c++11 -O3 -DNDEBUG src/*.cpp
```